documentation/book/proc-upgrading-the-cluster-operator-0-9-0-to-0-10-0.adoc

```diff
@@ -159,7 +153,7 @@ On {OpenShiftName}, use:
 oc get po my-cluster-kafka-0 -o jsonpath='{.spec.containers[0].image}'
 ----
 +
-The image tag shows the Strimzi version followed by the Kafka version: `{ProductVersionUpgradingTo}-kafka-{KafkaVersionLower}`.
+The image tag shows the Strimzi version followed by the Kafka version. For example, `{ProductVersionUpgradingTo}-kafka-{KafkaVersionLower}`.
```

Comment:
Strimzi should be replaced with the product attribute - not sure which one that is.

Reply:
Changed to {ProductName}. (This file isn't included downstream)

Reply:
We should still be consistent.